templates/HAPROXY_BACKEND_HTTP_HEALTHCHECK_OPTIONS

```diff
@@ -0,0 +1,2 @@
+  option  httpchk GET {healthCheckPath}
```

Comment:
So if we think this makes sense we should just make those defaults. Thoughts?
@brndnmtthws Maybe you can split out the `--health-check` changes and create a separate PR for that? IMO would make sense to have this in a separate commit anyway